config/tags.php

```diff
@@ -300,6 +301,7 @@
 				// for ex  : `autoplay` will not work if `false` is a `string` instead of a `boolean`
 				$attrs['autoplay']    = $autoplay = Str::toType($tag->autoplay, 'bool');
 				$attrs['controls']    = Str::toType($tag->controls ?? true, 'bool');
+				$attrs['disablepictureinpicture'] = Str::toType($tag->disablepictureinpicture ?? false, 'bool');
```

Comment:
Could you update the comment above or rewrite it in a way that doesn't list the individual attributes?